clang-tools-extra/docs/clang-tidy/checks/readability/math-missing-parentheses.rst

```diff
@@ -0,0 +1,20 @@
+.. title:: clang-tidy - readability-math-missing-parentheses
+
+readability-math-missing-parentheses
+====================================
+
+Check for mising parantheses in mathematical expressions that involve operators
+of different priorities.
+
+Before:
+
+.. code-block:: c++
+
+  int x = 1 + 2 * 3 - 4 / 5;
+
+
+After:
+
+.. code-block:: c++
+
+  int x = (1 + (2 * 3)) - (4 / 5);
```

Comment:
target should be:
`int x = 1 + (2 * 3) - (4 / 5);`

same like in example: 1 * 2 / 3 *10 / 23, no parentheses should be inserted.

Reply:
done